.ci/scripts/otel-update.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+set -euo pipefail
+
+usage() {
+  echo "Usage: $0 <next-beta-core> <next-stable-core> [<next-contrib>]"
+  echo "  <next-beta-core>: Next version of the unstable core components (e.g. v0.120.0). Get it from https://github.com/open-telemetry/opentelemetry-collector/releases."
+  echo "  <next-stable-core>: Next stable version of the stable core components (e.g. v1.26.0). Get it from https://github.com/open-telemetry/opentelemetry-collector/releases."
+  echo "  <next-contrib>: Next version of the contrib components (e.g. v0.120.1). Get it from https://github.com/open-telemetry/opentelemetry-collector-contrib/releases. If not specified, <next-beta-core> is used."
+  echo
+  exit 1
+}
+next_beta_core=${1:-}
+[[ -n "$next_beta_core" ]] || (echo "Error: missing <next-beta-core>" && echo && usage)
+
+next_stable_core=${2:-}
+[[ -n "$next_stable_core" ]] || (echo "Error: missing <next-stable-core>" && echo && usage)
+
+next_contrib=${3:-$next_beta_core}
+
+# Get current versions from go.mod
+current_beta_core=$(grep 'go\.opentelemetry\.io/collector/receiver/otlpreceiver ' go.mod | cut -d' ' -f 2)
+current_stable_core=$(grep 'go\.opentelemetry\.io/collector/confmap/provider/fileprovider ' go.mod | cut -d' ' -f 2)
+current_contrib=$(grep 'github\.com/open-telemetry/opentelemetry-collector-contrib/receiver/filelogreceiver ' go.mod | cut -d' ' -f 2)
```

Comment:
Could you add a condition to error out if any of these are empty? If for whatever reason these lines can't be found, we shouldn't proceed.